charts/hmc-hmi-outbound-adapter/values.yaml

```diff
@@ -9,13 +9,13 @@ java:
         - name: AppInsightsInstrumentationKey
           alias: azure.application-insights.instrumentation-key
         - name: ClientId
-          alias: fh.ad.client-id
+          alias: CLIENT_ID
         - name: ClientSecret
-          alias: fh.ad.client-secret
+          alias: CLIENT_SECRET
```

Comment:
secret names looks a  bit too generic, but minor thing anyway. can be changed later if need be.